types/easy-soap-request/index.d.ts

```diff
@@ -0,0 +1,71 @@
+// Type definitions for easy-soap-request 4.1
+// Project: https://github.com/circa10a/easy-soap-request
+// Definitions by: Steven Snoeijen <https://github.com/stevensnoeijen>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+import { AxiosProxyConfig, AxiosRequestConfig, AxiosResponse, AxiosError } from "axios";
+
+export interface Options {
+    /**
+     * HTTP request method
+     * @default 'POST'
+     */
+    method?: string;
+
+    /**
+     * endpoint URL
+     */
+    url: string;
+
+    /**
+     * HTTP headers, can be string or object
+     */
+    headers?: object | string;
+
+    /**
+     *  SOAP envelope, can be read from file or passed as string
+     */
+    xml: string;
+
+    /**
+     * Milliseconds before timing out request
+     * @default 10000
+     */
+    timeout?: number;
+
+    /**
+     * Object with proxy configuration
+     */
+    proxy?: AxiosProxyConfig;
+
+    /**
+     * Limit body size being sent(bytes)
+     * @default Infinity
+     */
+    maxBodyLength?: number;
+
+    /**
+     * Limit content size being sent(bytes)
+     * @default Infinity
+     */
+    maxContentLength?: number;
+
+    /**
+     * Object of additional axios parameters.
+     */
+    extraOpts?: AxiosRequestConfig;
+}
+
+export interface Response {
+    response: {
+        headers: any;
+        body: any;
+        statusCode: number;
+    };
+}
+
+/**
+ * @returns parsed from a AxiosResponse
+ * @throws {any|AxiosError} response from AxiosError.response.data
+ */
+export default function soapRequest(options: Options): Promise<Response>;
```

Comment:
This package was declared using `module.exports = ...` or some variant. It should be declared with an `export = ...` like one of the following:

To import the entity, you can use the following syntax:

```ts
import foo = require("foo");
```

[See more here in the FAQ](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/9f9f6430117d025aef62ed56128deb65508edc86/README.md#a-package-uses-export--but-i-prefer-to-use-default-imports-can-i-change-export--to-export-default).

```suggestion
/**
 * @returns parsed from a AxiosResponse
 * @throws {any|AxiosError} response from AxiosError.response.data
 */
function soapRequest(options: Options): Promise<Response>;

export = soapRequest
```